qutebrowser/completion/models/base.py

```diff
@@ -40,12 +40,18 @@ class BaseCompletionModel(QStandardItemModel):
     Used for showing completions later in the CompletionView. Supports setting
     marks and adding new categories/items easily.
 
+    Attributes:
+        dumb_sort:  the sorting that will be used by the wrapping
+                    CompletionFilterModel
+
```

Comment:
I think this can be removed, as there is no such attribute anymore (only the class attribute)